### DISCUSSION TITLE
Hi How long does project restoration takes?

### DISCUSSION THREAD

**Discussion by pushkarsingh32** [supabase/supabase#13776] [General]
I have setup authentication flow with supabase. Got busy with some other stuff. Now project is paused & it is not restoring from 5 hours. My db hardly have any data as it is in developing stage. 
Anyone can share any tips or help? 
Thanks

**Comment by fsansalvadore** [comment]
It usually takes no more than a few minutes to successfully restore a paused project.

In case your project gets stuck in “restoration” mode, the best option is to [contact the support team](https://app.supabase.com/support/new) specifying the project that’s facing the issue.

**Comment by wax911** [comment]
> I have setup authentication flow with supabase. Got busy with some other stuff. Now project is paused & it is not restoring from 5 hours. My db hardly have any data as it is in developing stage. Anyone can share any tips or help? Thanks

It's been weeks if not months for me, did you ever get any help on this @pushkarsingh32?

**Reply by pushkarsingh32** [reply]
Yes contacted support & they help me restore

**Reply by wax911** [reply]
Thanks! I'll do that then 😃

**Reply by kiwicopple** [reply]
(Maintainer) Restarting shouldn't take more than a couple of minutes (even less than 1 minute if your database is small). If it doesn't work, please reach out to support and we will check if there are any blockers

Also, don't worry - we backup all the data so your tables and old data is safe

**Reply by techfreakSahil** [reply]
@kiwicopple i restarted my project, and it's been more than 2 hour i'm stuck on the dashboard showing "restarting..."

**Reply by GaryAustin1** [reply]
@techfreakSahil Then you need to contact support.

**Reply by pushkarsingh32** [reply]
> @kiwicopple i restarted my project, and it's been more than 2 hour i'm stuck on the dashboard showing "restarting..."

Try to have a cron script which queries your database one time everyday & your job is solved : )

**Reply by roytmanpiccoli** [reply]
this company not give any support with have internal database error
